.travis.yml

```diff
@@ -26,14 +26,20 @@ matrix:
   include:
     - gemfile: gemfiles/Gemfile-rails-5-0
       rvm: 2.3.4
+    - gemfile: gemfiles/Gemfile-rails-5-1
+      rvm: 2.3.4
```

Comment:
We should also include Ruby 2.2 in this list as well since it's supported on Rails 5.0 and 5.1